proposals/lip-Introduce_cross_chain_messages.md

```diff
@@ -578,6 +580,17 @@ else:
 ```
 
 
+##### Executing Registration Message
```

Comment:
Don't we need to activate the channel here?

Reply:
The activation is actually done by the CCU execution. I updated the motivation and rationale to be more clear on this point.